src/types/org.ts

```diff
@@ -8,9 +8,27 @@ This file is part of @p0security/cli
 
 You should have received a copy of the GNU General Public License along with @p0security/cli. If not, see <https://www.gnu.org/licenses/>.
 **/
-import { bootstrapConfig } from "../drivers/env";
+type NonGoogleConfig = {
```

Comment:
nit: maybe we can call this something more descriptive like `ApplicationConfig`? calling it `NonGoogle` feels a bit strange.